cni/utils/errutils.go

```diff
@@ -1,15 +0,0 @@
-package utils
-
-import (
-	"github.com/Azure/azure-container-networking/cns/cnsclient"
-	"github.com/Azure/azure-container-networking/cns/restserver"
-)
-
-// TODO : Move to common directory like common, after fixing circular dependencies
```

Comment:
>ove to common directory like common, after fixing circular dependencies [](http://example.com/codeflow?start=11&length=71)

what is the circular dependency? @tamilmani1989 do you know?

Reply:
@rbtr correct me if im wrong. I think cns restserver package has dependency on cni common and this code cannot be moved to common because of that.  imo, this code shouldn't be in common and should be in cnsclient package. why didn't we put it there?

Reply:
@tamilmani1989, I deleted this file, I didn't write the comment so I don't know what the original reason for putting this here was. I have moved this functionality to the `cnsclient` package because that's where the `CNSClientError` struct is